### PR TITLE
Dark blue theme

### PR DESCRIPTION
**Pull request by envikas**

### CODE REVIEW

File: src/actions/types.js [hunk 1]
```diff
@@ -3,4 +3,4 @@ export const LOADING = 'LOADING';
 export const ELEMENT_SELECTED = 'ELEMENT_SELECTED';
 export const ELEMENT_REMOVED = 'ELEMENT_REMOVED';
 export const SERVER_ERROR = 'SERVER_ERROR';
-export const SERVER_URL = 'http://localhost:3004/';
+export const SERVER_URL = 'http://192.168.57.78:3004/';
```
Comment (aarora11) [inline src/actions/types.js]: Should be changed back to localhost

File: src/components/QueryBuilder.js [hunk 2]
```diff
@@ -20,7 +19,8 @@ class QueryBuilder extends Component{
     this.props.dropElement(ev.dragData.name);
   }
 
-  sendNameOFClickedTerm (name) {
+  sendNameOFClickedTerm (name, e) {
+    console.log(name);
```
Comment (aarora11) [inline src/components/QueryBuilder.js]: remove the console.log

File: src/components/Results.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import React, {Component} from 'react';
```
Comment (aarora11) [inline src/components/Results.js]: File needs to be deleted. This has been replaced by ResultGrid

File: src/components/App.js [hunk 4]
```diff
@@ -20,13 +23,14 @@ class App extends Component {
...
-          <div className="filters">
-            <p>Placeholder for the filters section</p>
+            <div className="filters">
+              <p>Placeholder for the filters section</p>
+            </div>
           </div>
           <div className="flex-container-result">
             <ResultsGrid results={this.props.results}/>
           </div>
+          <ConceptMenu/>
```
Comment (aarora11) [inline src/components/App.js]: Move <ConceptMenu/> into the QueryBuilder.js. This should not be included in the top level component.

File: src/components/ConceptMenu.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      //If you wrap more than one component, ref will be an array of ref
+      console.log(ref);
+      // Additionnal data props passed down to the `Item`
+      console.log(data);
+  }
+
+  render() {
+    return (
+      <ContextMenu id='menu_id'>
+          <Item leftIcon={<IconFont className="fa fa-plus" />} onClick={this.onClick}>
```
Comment (aarora11) [inline src/components/ConceptMenu.js]: <IconFont> is not implemented. Should be removed.

File: src/components/ConceptMenu.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+              Add
+          </Item>
+          <Item leftIcon={<IconFont className="material-icons">remove_circle</IconFont>} onClick={this.onClick}>
```
Comment (aarora11) [inline src/components/ConceptMenu.js]: Should only have two <Items>
1. Add filters.
2. Remove. 

On clicking add filters, a modal should pop up (already done in the branch titled modal)
On clicking remove, existing functionality to remove concept from the query builder should be triggered.

File: src/index.js [hunk 7]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './css/index.css';
 import 'bootstrap/dist/css/bootstrap.css';
+import 'react-contexify/dist/ReactContexify.min.css'
```
Comment (aarora11) [inline src/index.js]: Why do we need this? The default css provided doesn't integrate well with our look and feel.

File: src/actions/types.js (hunk 1 above)
Comment (ttrimbonias) [reply src/actions/types.js]: fixed

File: src/components/App.js (hunk 4 above)
Comment (ttrimbonias) [reply src/components/App.js]: fixed

File: src/components/ConceptMenu.js (hunk 5 above)
Comment (ttrimbonias) [reply src/components/ConceptMenu.js]: fixed

File: src/components/ConceptMenu.js (hunk 6 above)
Comment (ttrimbonias) [reply src/components/ConceptMenu.js]: Changed to 2.  Implementation to be merged later

File: src/components/QueryBuilder.js (hunk 2 above)
Comment (ttrimbonias) [reply src/components/QueryBuilder.js]: fixed

File: src/components/Results.js (hunk 3 above)
Comment (ttrimbonias) [reply src/components/Results.js]: fixed

File: src/index.js (hunk 7 above)
Comment (ttrimbonias) [reply src/index.js]: won't work without it

**Pull request merged**